src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public abstract class Machine {
+
+    public abstract void doWork();
```

Comment:
```suggestion
public abstract class Machine {
    public abstract void doWork();
```
![image](https://github.com/mate-academy/jv-oop/assets/20326858/36576c91-e67b-4825-ac9d-099bfaf60ae4)